mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
+    .catch(error => { 
+      res.status(500).json({
+        error: errorHandler(error),
+      });
```

Comment:
This code snippet appears to be a closing bracket of a function or a block of code, but it is not clear what it is closing without more context. 

Please provide more context or the entire function/block of code to review properly.